src/test/java/org/mockitousage/matchers/MoreMatchersTest.java

```diff
@@ -48,12 +48,14 @@ public void any_class_should_be_actual_alias_to_isA() {
         mock.simpleMethod((String) null);
         try {
             verify(mock).simpleMethod(isA(String.class));
-            fail();
-        } catch (AssertionError ignored) { }
-        try {
-            verify(mock).simpleMethod(any(String.class));
-            fail();
-        } catch (AssertionError ignored) { }
+        } catch (AssertionError ignored) {
```

Comment:
The resulting code is less readable. I suggest AssertJ or suppression.

Reply:
done. Will look nicer once we can use lambdas